### DISCUSSION TITLE
Should there be more specific schema types

### DISCUSSION THREAD

**Discussion by Gerhardsa0** [Safe-DS/.github#7] [Library Discussion]
Right now, there only exists one type of schema, which represents a general tabular schema. But there are more types of tables in the Safe-DS Library, so I thought of a schema-concept which gets more specific for more specific tabular. 
So should there be something like a time series schema or tagged-table schema?

**Comment by lars-reimann** [comment]
Wrt. to its schema, a TaggedTable is identical to a Table. For a TimeSeries, I don't see the need for an extra kind of schema either. Instead, we should add some temporal column types to the existing schema description.

**Reply by Gerhardsa0** [reply]
Ah, okay, so we have one schema which can describe the table more precise. So there should be columns like: target and feature column as a Type for a schema.

**Reply by lars-reimann** [reply]
A schema assigns datatypes to columns, not roles.
